src/test/java/org/toradocu/PrecisionRecallCommonsMath3.java

```diff
@@ -47,7 +54,7 @@ public void testPrimes() throws Exception {
 
   @Test
   public void testBitsStreamGenerator() throws Exception {
-    test("org.apache.commons.math3.random.BitsStreamGenerator", 1, 0.5, 1, 0.8, 1, 1);
+    test("org.apache.commons.math3.random.BitsStreamGenerator", 0.5, 0.5, 1, 0.8, 1, 1);
```

Comment:
Similar cases to the one above. Here we have 2 composite conditions, before empty, now partially translated - but not completely, thus, wrong condition.